tasks/find-replace.js

```
@@ -66,25 +69,60 @@ const getFullTaskList = (options) => {
+            title: 'Reporting matches',
+            enabled: () => dryRun,
+            task: async (ctx, task) => {
+                if (ctx.toUpdate.length === 0) {
+                    task.title = 'No matches found';
+                    return;
+                }
+
+                let totalMatches = 0;
+
+                for (const post of ctx.toUpdate) {
+                    for (const [field, count] of Object.entries(post.matchesByField)) {
```

ESLint

[error] 'field' is assigned a value but never used.

(no-unused-vars)